conversationdemo/conversationdemo/CallKit/ProviderDelegate.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ProviderDelegate.swift
+//  conversationdemo
+//
+//  Created by Eric Giannini on 8/14/18.
+//  Copyright © 2018 Nexmo, Inc. All rights reserved.
+//
+
+import AVFoundation
+import CallKit
+
+class ProviderDelegate: NSObject {
+    
+    fileprivate let callManager: CallManager
+    fileprivate let provider: CXProvider
+    
+    init(callManager: CallManager) {
+        self.callManager = callManager
+        provider = CXProvider(configuration: type(of: self).providerConfiguration)
+        super.init()
+        provider.setDelegate(self, queue: nil)
+    }
+    
+    static var providerConfiguration: CXProviderConfiguration {
+        let providerConfiguration = CXProviderConfiguration(localizedName: "Stitch")
+        providerConfiguration.supportsVideo = false
+        providerConfiguration.maximumCallsPerCallGroup = 1
+        providerConfiguration.supportedHandleTypes = [.phoneNumber]
+        
+        return providerConfiguration
+    }
+    
+    func reportIncomingCall(uuid: UUID, handle: String, hasVideo: Bool = false, completion: ((NSError?) -> Void)?) {
```

Comment:
`NSError` should be changed to `Error`, since your casting it from `Error` back to `NSError`